website/components/mdx/_download-linux.mdx

```diff
@@ -5,7 +5,7 @@ import { Tabs, Tab } from '../Tabs'
 <Tab>
 
 ```bash copy
-curl -L https://versions.cloudquery.io/latest/v2/cloudquery_linux_amd64 -o cloudquery
+curl -L https://github.com/cloudquery/cloudquery/releases/download/VERSION_CLI/cloudquery_linux_amd64 -o cloudquery
```

Comment:
Maybe we should keep the latest link too? Otherwise there's no easy way to download CloudQuery latest version (without looking in the docs)